frontend/packages/client/src/pages/Community.js

```diff
@@ -335,13 +360,29 @@ export default function Community() {
             <div className="columns m-0 p-0">
               <div className="column p-0">
                 <div className="tabs tabs-community is-medium small-text">
-                  <ul className="tabs-community-list">
+                  <ul
+                    ref={wrapperRef}
+                    className="tabs-community-list"
+                    style={{ position: 'relative' }}
+                  >
+                    <div
+                      style={{
+                        transition: '0.1s all',
+                        position: 'absolute',
+                        bottom: 0,
+                        height: 2,
+                        backgroundColor: '#00EF8B',
+                        width: activeWidth,
+                        left: activeLeft,
+                      }}
+                    />
```

Comment:
this is our underline element that transitions its width and left offset smoothly